docker-compose.yml

```
@@ -52,12 +52,41 @@ services:
       - "1080:1080"
       - "1025:1025"
 
+  zookeeper:
+    image: confluentinc/cp-zookeeper:latest
+    container_name: ms_zookeeper
+    environment:
+      ZOOKEEPER_SERVER_ID: 1
+      ZOOKEEPER_CLIENT_PORT: 2181
+      ZOOKEEPER_TICK_TIME: 2000
+    ports:
+        - "2181:2181"
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 8

(indentation)